src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+function createNotification(messange, type) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = type; // // success або error
+  notification.textContent = messange;
+  document.body.appendChild(notification);
+}
+
+// firstPromise: вирішується або відхиляється через 3 секунди
+let firstPromiseResolved = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    if (!firstPromiseResolved) {
+      firstPromiseResolved = true;
+      resolve('First promise was resolved');
+    }
+  });
+
+  // відхилення через 3 с,якщо не клацнути
+  setTimeout(() => {
+    if (!firstPromiseResolved) {
+      resolve(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+// обробник для firsPromise
+firstPromise
+  .then((messange) => createNotification(messange, 'success'))
```

Comment:
The variable `messange` is used here, which contains a typo. It should be `message` to match the intended parameter name.